### ISSUE TITLE
I can not activate the framwork

### ISSUE THREAD

**Issue by partharoy** [vafour/vafpress-framework#99]
Hi, 
I have tried to activate your plugin but failed. I implied this code require_once 'vafpress/bootstrap.php';
 But it does not work. Is there another process to activate it?

**Comment by Zackio** [comment]
Did you get an error? You just need to include this but also you have to set up the data

http://vafpress.com/documentation/vafpress-framework//getting-started.html

**Comment by partharoy** [comment]
no. no errors comes up. but is not it show a admin bar like theme option?

**Comment by Zackio** [comment]
I moved mine from the default location, but if i recall correctly i think
it's meant to come under apparence

**Comment by partharoy** [comment]
yes I know. but there is no theme option under appearance nor admin bar.

**Comment by Zackio** [comment]
You've used the code at the top of this page?

http://vafpress.com/documentation/vafpress-framework//options/builder.html

**Comment by partharoy** [comment]
no at the bottom of the functions.php but after see your comments I have tried it if first under <?php code. but nothing happen

**Comment by Zackio** [comment]
this is what i have, which works for me

include THEME_FRAMEWORK_DIR . 'importer/demo_options_import.php';

$theme_options_file = get_template_directory() . '/framework/option.php';
global $theme_option_panel;
$theme_option_panel = new theme_demo_options( array(
        'is_dev_mode'           => false,
        'option_key'            => 'theme_options',
        'page_slug'             => 'op',
        'template'              => $theme_options_file,
        'menu_page' => array(
            'icon' => 'font-awesome:fa-star',
        ),
        'use_auto_group_naming' => true,
        'use_exim_menu'         => true,
        'minimum_role'          => 'edit_theme_options',
        'layout'                => 'fixed',
        'page_title'            => __( 'Theme Options', 'VP_TEXTDOMAIN' ),
        'menu_label'            => __( 'Theme Options', 'VP_TEXTDOMAIN' ),
    ) );

**Comment by TheZoker** [comment]
Look at this example:
https://github.com/vafour/vafpress-framework-theme-boilerplate/blob/master/functions.php
